src/e2e-test/java/io/cdap/plugin/pubsub/locators/PubSubLocators.java

```diff
@@ -64,6 +64,20 @@ public class PubSubLocators {
 
   @FindBy(how = How.XPATH, using = "//input[@data-cy='numberOfReaders']")
   public static WebElement numberOfReaders;
+  @FindBy(how = How.XPATH, using = "//input[@placeholder='Field name' and @value='']")
+  public static WebElement addField;
+  @FindBy(how = How.XPATH, using = "//span[contains(text(), \"Batch interval\")]//following-sibling::div//select[1]")
+  public static WebElement batchTime;
+  @FindBy(how = How.XPATH, using = "//span[contains(text(), \"Batch interval\")]//following-sibling::div//select[2]")
+  public static WebElement timeSelect;
```

Comment:
Try writing better locators for time select and batch time if possible.

Reply:
These locators are referenced from salesforce runtime code.